packages/core/src/internal/index.ts

```diff
@@ -42,4 +45,13 @@ export * from './utils/supports.js';
 export * from './utils/events.js';
 export * from './utils/event-subject.js';
 export * from './interfaces/index.js';
-export { styles as baseStyles } from './base/base.element.css.js';
+export * from './motion/interfaces.js';
+export * from './motion/motion.service.js';
+export * from './motion/utils.js';
+export { AnimationModalEnterConfig, AnimationModalEnterName } from './motion/animations/cds-modal-enter.js';
+export {
+  AnimationAccordionPanelOpenConfig,
+  AnimationAccordionPanelOpenName,
+} from './motion/animations/cds-accordion-panel-open.js';
+export { AnimationHingeConfig, AnimationHingeName } from './motion/animations/cds-overlay-hinge-example.js';
+export { AnimationShakeConfig, AnimationShakeName } from './motion/animations/cds-component-shake.js';
```

Comment:
Rollup does not allow deep barrel imports `index.js` which points to another `index.js` since it can cause treeshaking issues. So the fix was to remove the inner index and move the exports to the `internal/index.js`